SpeckleRhinoPlugin/SpeckleMacR5/SpeckleMacR5.csproj

```diff
@@ -32,19 +32,18 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="..\src\EtoForm.cs">
-      <Link>EtoForm.cs</Link>
-    </Compile>
   </ItemGroup>
   <ItemGroup>
-    <Content Include="..\app\index.html">
-      <Link>app\index.html</Link>
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </Content>
-    <Content Include="..\app\app.js">
-      <Link>app\app.js</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
+    <PackageReference Include="Newtonsoft.Json">
+      <Version>11.0.1</Version>
+    </PackageReference>
+    <PackageReference Include="RhinoCommon">
+      <Version>5.12.50810.13095</Version>
+    </PackageReference>
   </ItemGroup>
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
+  <Target Name="AfterBuild">
+    <Copy SourceFiles="$(TargetPath)" DestinationFiles="$(TargetDir)\$(ProjectName).rhp" />
```

Comment:
@fraguada This could potentially be handled by adding a `TargetExt` property. See https://github.com/mcneel/rhino/blob/6.x/src4/RhinoProjectPropertySheets/Rhino.CS.PlugIn.props#L17...